packages/valory/skills/apy_estimation_abci/tests/test_behaviours.py

```diff
@@ -407,6 +413,8 @@ def test_retries_exceeded(
         # we expect `retries_exceeded` to return True if any of the subgraphs' `is_retries_exceeded` returns `True`
         expected = any(is_exceeded_per_subgraph)
         assert behaviour.retries_exceeded == expected
+        behaviour._utilized_subgraphs = {}
```

Comment:
Reset `_utilized_subgraphs` after the check.

Reply:
Also, generally would do this on the teardown or setup so it's very easy to see all the conditions in one place.